src/elements/content-explorer/ItemGridThumbnail.scss

```diff
@@ -11,10 +15,16 @@
     background-repeat: no-repeat;
     background-size: contain;
     bottom: 0;
+    cursor: pointer;
     display: flex;
     justify-content: center;
     left: 0;
     position: absolute;
     right: 0;
     top: 0;
+
+    & svg {
+        height: 60%;
+        width: 60%;
```

Comment:
This lets generic BoxIcons scale as the cards change size.  The height and width should be bounded by the grid card itself, so is using `%'s` here ok?